src/legacy/status_im/data_store/settings.cljs

```diff
@@ -26,7 +34,7 @@
       (update :wallet-legacy/visible-tokens rpc->visible-tokens)
       (update :pinned-mailservers rpc->pinned-mailservers)
       (update :link-previews-enabled-sites set)
-      (update :currency keyword)
+      (update :currency rpc->currency)
```

Comment:
we need to apply the same in the `rpc->setting-value` method below this.

Reply:
Additionally, in this place as well where we update from signal

https://github.com/status-im/status-mobile/blob/43651ef0d0020eeea829bad437a6769427225cfe/src/legacy/status_im/multiaccounts/update/core.cljs#L84